src/rules/functionality.ts

```diff
@@ -51,6 +59,10 @@ export default {
   'no-string-throw': true,
   // Disallows falling through case statements.
   'no-switch-case-fall-through': true,
+   // Warns when a method is used as outside of a method call.
+  'no-unbound-method': false,
```

Comment:
This rule's already part of our [`untyped config`](https://github.com/Shopify/tslint-config-shopify/blob/master/src/untyped.ts#L15).
I believe when you were digging through the [tslint](https://github.com/palantir/tslint) repo to compile the list of rules requiring type-checking, you managed to pick up the new rules that we're integrating now.